integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/AbstractServletContainerIntegrationTest.java

```diff
@@ -81,11 +81,22 @@
  * </p>
  */
 public abstract class AbstractServletContainerIntegrationTest {
+    private static final Logger logger = LoggerFactory.getLogger(AbstractServletContainerIntegrationTest.class);
+
     private static final String pathToJavaagent;
     private static final String pathToAttach;
-    private static final Logger logger = LoggerFactory.getLogger(AbstractServletContainerIntegrationTest.class);
+    private static final String pathToSlimAttach;
+
     static boolean ENABLE_DEBUGGING = false;
     static boolean ENABLE_RUNTIME_ATTACH = true;
+
+    /**
+     * Set to a specific version to test downloading of agent from maven central using the slim cli tool.
+     * Only relevant if {@link #ENABLE_RUNTIME_ATTACH} is set to {@code true}.
+     */
+    @Nullable
+    private static final String AGENT_VERSION_TO_DOWNLOAD_FROM_MAVEN = null;
```

Comment:
[question] If I understand correctly, this is only set when testing manually, just like the `ENABLE_DEBUGGING` is used to help when debugging issues. If yes, then adding a comment to make that 100% obvious could be relevant.

Reply:
Yes, you're right